pages/Addons.md

```diff
@@ -64,6 +64,7 @@ SlimeTinker | Sefiraat | <details><summary>expand</summary>SlimeTinker is a Slim
 PotionExpansion | EpicPlayerA10 | <details><summary>expand</summary>An addon which adds new potions that give you legal xray!</details> | [Readme](https://github.com/EpicPlayerA10/PotionExpansion/blob/master/README.md) | [GitHub](https://github.com/EpicPlayerA10/PotionExpansion) | [Builds](https://thebusybiscuit.github.io/builds/EpicPlayerA10/PotionExpansion/master/)
 FlowerPower | NCBPFluffyBear | <details><summary>expand</summary>FlowerPower is a magic addon centered around experience and flowers. The base of each recipe requires various flowers, and different items are crafted along the way to make mid to endgame items. A majority of these items function by consuming experience in exchange for a certain effect or buff.</details> | [Readme](https://github.com/NCBPFluffyBear/FlowerPower/blob/master/README.md) | [GitHub](https://github.com/NCBPFluffyBear/FlowerPower/) | [Builds](https://thebusybiscuit.github.io/builds/NCBPFluffyBear/FlowerPower/master/)
 Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand</summary>Galactifun is an addon inspired by ClayTech and the mod Galacticraft. It adds multiple planets, rockets, space stations, and more!</details> | [Readme](https://github.com/Slimefun-Addon-Community/Galactifun/blob/master/README.md) | [GitHub](https://github.com/Slimefun-Addon-Community/Galactifun/) | [Builds](https://thebusybiscuit.github.io/builds/Slimefun-Addon-Community/Galactifun/master/)
+FNAmplifications | FN_FAL113 | <details><summary>expand</summary>4 in 1 addon that includes my FN Solar Generator, Material Generator, Machinery and Power Xpansion Addons which are based from the addons made by the slimefun addon community.</details> | [Readme](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/blob/main/README.md) | [GitHub](https://github.com/FN-FAL113/FN-FAL-s-Amplifications) | [Releases](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/releases)
```

Comment:
This way we dont start the sentence with a number which is a bit cleaner,
I could even see it going in the `A four in one` way over 4 in 1
```suggestion
FNAmplifications | FN_FAL113 | <details><summary>expand</summary>A 4 in 1 addon that includes my FN Solar Generator, Material Generator, Machinery and Power Xpansion Addons which are based from the addons made by the slimefun addon community.</details> | [Readme](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/blob/main/README.md) | [GitHub](https://github.com/FN-FAL113/FN-FAL-s-Amplifications) | [Releases](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/releases)
```

Reply:
Will change, thank you for that!